data/json/obsoletion_and_migration_0.I/migrated_vehicleparts.json

```diff
@@ -10,5 +10,17 @@
     "//": "Remove after 0.I release",
     "from": "ap_freezer",
     "to": "ap_chest_freezer"
+  },
+  {
+    "type": "vehicle_part_migration",
+    "//": "Remove after 0.I release",
+    "from": "car_light_plus_battery_cell",
+    "to": "battery_car"
```

Comment:
`car_light_plus_battery_cell` has `"location": "on_battery_mount"` which is not same as `battery_car`'s location.
It is possible to install tool battery mount, tool battery and car. battery on same tile In this situation migration will result in 2 car batteries on same tile. I do not know if that can cause problems.
Migrating it to `light_cell_rechargeable` instead would prevent those problems, if they exist. And players will not be annoyed when they discovered their AA battery they put in tool mount to be charged suddenly turned into 20kg lead-acid car battery

Reply:
it would affect only vehicle parts, and i don't think it's a big deal what player would get, as long as it does not error that "vehiclepart do not exist"